GabrielaRamirez/EjerciciosHackerrank/src/hackkerank/day1/IfElse.java

```diff
@@ -0,0 +1,29 @@
+package hackkerank.day1;
+import java.util.Scanner;
+
+public class IfElse {
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public static void main(String[] args) {
+        int N = scanner.nextInt();
+        scanner.skip("(\r\n|[\n\r\u2028\u2029\u0085])?");
+
+        scanner.close();
+
+        if (N%2==0 & 2<=N & N<=5){
```

Comment:
Why & ?? you mean && right ?? You know the difference between them?

2<=N ??? you mean N>=2 right?? Same case for 6<=N

Lines 13 and 16 are identical !! I think that's a mistake